src/lib/components/Preset.svelte

```diff
@@ -141,6 +141,13 @@
     C --> db
     D --> C
     style m fill:#d6d,stroke:#333,stroke-width:4px
+    `,
+    ZenUML: `zenuml
+    title Sync message
+    A.SyncMessage
+    A.SyncMessage(with, parameters) {
+      B.nestedSyncMessage()
+    }
     `
```

Comment:
It'll be better if you can add a bit more complex diagram, so people can see what all is possible.